api/python/cellxgene_census/src/cellxgene_census/experimental/ml/pytorch.py

```diff
@@ -300,8 +337,40 @@ def __init__(
         """
         Construct a new ``ExperimentDataPipe``.
 
+        Args:
+            experiment:
+                The SOMA ``Experiment`` from which to read data.
+            measurement_name:
+                The name of the SOMA ``Measurement`` to read. Defaults to "raw".
+            X_name:
+                The name of the X layer to read. Defaults to "X".
+            obs_query:
+                The query used to filter along the ``obs`` axis. If not specified, all ``obs`` and ``X`` data will
+                be returned, which can be very large.
+            var_query:
+                The query used to filter along the ``var`` axis. If not specified, all ``var`` columns (genes/features)
+                will be returned.
+            obs_column_names:
+                The names of the ``obs`` columns to return. The ``soma_joinid`` index "column" does not need to be
+                specified and will always be returned. If not specified, only the ``soma_joinid`` will be returned.
+            batch_size:
+                The number of rows of ``obs`` and ``X`` data to return in each iteration. Defaults to 1. A value of 1
+                will result in Tensors of rank 1 being returns (a single row); larger values will result in Tensors of
+                rank 2 (multiple rows).
+            sparse_X:
+                Controls whether the ``X`` data is returned as a dense or sparse Tensor. As ``X`` data is very sparse,
+                setting this to ``True`` will reduce memory usage, if the model supports use of sparse Tensors. Defaults
+                to ``False``, since sparse Tensors are still experimental in PyTorch.
+            soma_buffer_bytes:
+                The number of bytes to use for reading data from SOMA. If not specified, will use the default SOMA
+                value. Maximum memory utilization is controlled by this parameter, with larger values providing better
+                read performance.
+            use_eager_fetch:
+                Controls whether the returned iterator will eagerly fetch data from SOMA while client code is iterating
+                through data. Defaults to ``True``.
+
         Returns:
-            ``ExperimentDataPipe``.
+            ``ExperimentDataPipe``
```

Comment:
Even if this is a bit redundant, this is generally expressed as a sentence, so maybe
```suggestion
            The constructed ``ExperimentDataPipe``.
```

Reply:
done! (made change along with some others)